src/ImageSharp/Formats/Png/PngDecoderCore.cs

```diff
@@ -264,6 +329,14 @@ public ImageInfo Identify(BufferedReadStream stream, CancellationToken cancellat
                         case PngChunkType.Header:
                             this.ReadHeaderChunk(pngMetadata, chunk.Data.GetSpan());
                             break;
+                        case PngChunkType.AnimationControl:
```

Comment:
When decoding multi-frame images the decoder should respect the `DecoderOptions.MaxFrames` property.